radar-auth-jersey/src/main/kotlin/org/radarbase/auth/jersey/AuthConfig.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2019. The Hyve
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * See the file LICENSE in the root of this repository.
+ */
+
+package org.radarbase.auth.jersey
+
+data class AuthConfig(
+        val managementPortalUrl: String? = null,
+        val jwtResourceName: String,
+        val jwtIssuer: String? = null,
+        val jwtECPublicKeys: List<String>? = null,
+        val jwtRSAPublicKeys: List<String>? = null,
+        val jwtKeystorePath: String? = null,
```

Comment:
Should we continue to support key-store related configs here, since after MP:0.5.4 key-store is related things are handled by MP. Having a config for retrieving JWK should sufficient, if I am not mistaken. Since It is a new library, it would be good to eliminate legacy configs.

Reply:
These configs are only used if a raw ECDSA provider like Keycloak is used.